src/hotspot/share/opto/doCall.cpp

```diff
@@ -1188,11 +1188,6 @@ ciMethod* Compile::optimize_inlining(ciMethod* caller, int bci, ciInstanceKlass*
     // such method can be changed when its class is redefined.
     ciMethod* exact_method = callee->resolve_invoke(calling_klass, actual_receiver);
     if (exact_method != NULL) {
-      if (PrintOpto) {
-        tty->print("  Calling method via exact type @%d --- ", bci);
-        exact_method->print_name();
-        tty->cr();
-      }
```

Comment:
Why remove this?

Reply:
In upstream, JDK-8257211  removes argument bci and this log printing. We didn't backport JDK-8257211, so we remove it in Vtable-based CHA in order to keep consistent with upstream. It might be better to keep it.